tests/unit/core/utils/test_index.js

```diff
@@ -404,6 +404,21 @@ describe(__filename, () => {
     it('handles null text', () => {
       expect(nl2br(null)).toEqual('');
     });
+
+    it('returns mixed content with <br/>', () => {
+      expect(nl2br('foo\nbar\n\n<b>bold</b>')).toEqual(
+        'foo<br />bar<br />\n<b>bold</b>',
+      );
+    });
+
+    it('returns valid HTML', () => {
+      const htmlValue =
+        '<ul>\n<li>ul\nli<strong>foo\nbar</strong>\n</li>\n</ul>';
```

Comment:
There is a lot going on in this test:
- line breaks between tags are preserved (multiple instances)
- line breaks in tag content are converted (multiple instances)
- tag content that looks like tags do not trick the converter

It would be easier to read the failures if you broke it down into a few tests, each focusing on a single case.